src/main/java/io/jenkins/plugins/analysis/core/restapi/ReportApi.java

```diff
@@ -33,7 +40,14 @@ public ReportApi(final Report report) {
     }
 
     private List<IssueApi> map() {
-        return report.stream().map(IssueApi::new).collect(Collectors.toList());
+        return report.stream().map(this::createIssueApi).collect(Collectors.toList());
+    }
+
+    private IssueApi createIssueApi(Issue issue) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'issue' is not assigned and could be declared final](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5045840)